.github/workflows/ci.yml

```diff
@@ -247,22 +252,24 @@ jobs:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
           password: ${{ secrets.CR_PAT }}
+        if: ${{ github.event_name == 'push' }}
 
       - name: Login to Docker Hub
         uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+        if: ${{ github.event_name == 'push' }}
```

Comment:
**Question**: why don't we need to log in when we are pushing a tag image for example?

Reply:
We only push images on a push event.